2023-02-27 23:44:37.516 -05:00 [INF] User profile is available. Using 'C:\Users\MonPC\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2023-02-27 23:44:42.946 -05:00 [INF] Application started. Press Ctrl+C to shut down.
2023-02-27 23:44:42.979 -05:00 [INF] Hosting environment: Development
2023-02-27 23:44:42.980 -05:00 [INF] Content root path: C:\Users\MonPC\source\repos\allopromo\allopromo.Api
2023-02-27 23:44:43.122 -05:00 [INF] Request starting HTTP/2.0 GET https://localhost:44306/swagger/index.html  
2023-02-27 23:44:44.243 -05:00 [INF] Request finished in 1137.199ms 200 text/html;charset=utf-8
2023-02-27 23:44:47.858 -05:00 [INF] Request starting HTTP/2.0 GET https://localhost:44306/swagger/v1/swagger.json  
2023-02-27 23:44:51.168 -05:00 [INF] Request finished in 3310.3503ms 200 application/json;charset=utf-8
2023-02-27 23:45:43.527 -05:00 [INF] Request starting HTTP/2.0 POST https://localhost:44306/api/v1/User/register application/json-patch+json 151
2023-02-27 23:45:43.647 -05:00 [INF] CORS policy execution successful.
2023-02-27 23:45:43.653 -05:00 [INF] Executing endpoint 'allopromo.Controllers.UserController.CreateUser (allopromo.Api)'
2023-02-27 23:45:44.056 -05:00 [INF] Route matched with {action = "CreateUser", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] CreateUser(allopromo.Api.ViewModel.ViewModels.RegisterViewModel) on controller allopromo.Controllers.UserController (allopromo.Api).
2023-02-27 23:45:53.047 -05:00 [INF] Entity Framework Core 5.0.7 initialized 'AppDbContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None
2023-02-27 23:45:59.861 -05:00 [INF] Executed DbCommand (1,557ms) [Parameters=[@__normalizedName_0='?' (Size = 256)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [a].[Id], [a].[ConcurrencyStamp], [a].[Name], [a].[NormalizedName]
FROM [AspNetRoles] AS [a]
WHERE [a].[NormalizedName] = @__normalizedName_0
2023-02-27 23:46:00.122 -05:00 [INF] Executed DbCommand (49ms) [Parameters=[@__normalizedName_0='?' (Size = 256)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [a].[Id], [a].[ConcurrencyStamp], [a].[Name], [a].[NormalizedName]
FROM [AspNetRoles] AS [a]
WHERE [a].[NormalizedName] = @__normalizedName_0
2023-02-27 23:46:03.279 -05:00 [INF] Executed DbCommand (1,771ms) [Parameters=[@p0='?' (Size = 450), @p1='?' (Size = 4000), @p2='?' (Size = 256), @p3='?' (Size = 256)], CommandType='"Text"', CommandTimeout='30']
SET NOCOUNT ON;
INSERT INTO [AspNetRoles] ([Id], [ConcurrencyStamp], [Name], [NormalizedName])
VALUES (@p0, @p1, @p2, @p3);
2023-02-27 23:46:16.760 -05:00 [INF] Executing ObjectResult, writing value of type 'allopromo.Core.Domain.ApplicationUser'.
2023-02-27 23:46:17.001 -05:00 [INF] Executed DbCommand (321ms) [Parameters=[@__normalizedRoleName_0='?' (Size = 256)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(2) [a].[Id], [a].[ConcurrencyStamp], [a].[Name], [a].[NormalizedName]
FROM [AspNetRoles] AS [a]
WHERE [a].[NormalizedName] = @__normalizedRoleName_0
2023-02-27 23:46:17.245 -05:00 [INF] Executed action allopromo.Controllers.UserController.CreateUser (allopromo.Api) in 33144.3287ms
2023-02-27 23:46:17.252 -05:00 [INF] Executed endpoint 'allopromo.Controllers.UserController.CreateUser (allopromo.Api)'
2023-02-27 23:46:18.442 -05:00 [INF] Request finished in 35139.5033ms 200 application/json; charset=utf-8
2023-02-27 23:46:19.816 -05:00 [ERR] Failed executing DbCommand (2,535ms) [Parameters=[@__p_0='?' (Size = 450), @__p_1='?' (Size = 450)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [a].[UserId], [a].[RoleId], [a].[Discriminator], [a].[RoleId1], [a].[UserId1]
FROM [AspNetUserRoles] AS [a]
WHERE ([a].[UserId] = @__p_0) AND ([a].[RoleId] = @__p_1)
2023-02-27 23:46:21.485 -05:00 [ERR] An exception occurred while iterating over the results of a query for context type 'allopromo.Infrastructure.Data.AppDbContext'.
System.InvalidOperationException: Invalid operation. The connection is closed.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__169_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
System.InvalidOperationException: Invalid operation. The connection is closed.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__169_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2023-02-27 23:51:24.591 -05:00 [INF] Request starting HTTP/2.0 POST https://localhost:44306/api/v1/User/register application/json-patch+json 151
2023-02-27 23:51:24.612 -05:00 [INF] CORS policy execution successful.
2023-02-27 23:51:24.615 -05:00 [INF] Executing endpoint 'allopromo.Controllers.UserController.CreateUser (allopromo.Api)'
2023-02-27 23:51:24.657 -05:00 [INF] Route matched with {action = "CreateUser", controller = "User"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] CreateUser(allopromo.Api.ViewModel.ViewModels.RegisterViewModel) on controller allopromo.Controllers.UserController (allopromo.Api).
2023-02-27 23:51:40.712 -05:00 [INF] Entity Framework Core 5.0.7 initialized 'AppDbContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None
2023-02-27 23:51:41.761 -05:00 [INF] Executed DbCommand (340ms) [Parameters=[@__normalizedName_0='?' (Size = 256)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [a].[Id], [a].[ConcurrencyStamp], [a].[Name], [a].[NormalizedName]
FROM [AspNetRoles] AS [a]
WHERE [a].[NormalizedName] = @__normalizedName_0
2023-02-27 23:51:41.905 -05:00 [INF] Executed DbCommand (81ms) [Parameters=[@__normalizedName_0='?' (Size = 256)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [a].[Id], [a].[ConcurrencyStamp], [a].[Name], [a].[NormalizedName]
FROM [AspNetRoles] AS [a]
WHERE [a].[NormalizedName] = @__normalizedName_0
2023-02-27 23:51:41.924 -05:00 [WRN] Role 927d5ce4-e31e-457f-9617-ad458552887d validation failed: DuplicateRoleName;DuplicateRoleName.
2023-02-27 23:51:49.835 -05:00 [INF] Executed DbCommand (39ms) [Parameters=[@__normalizedRoleName_0='?' (Size = 256)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(2) [a].[Id], [a].[ConcurrencyStamp], [a].[Name], [a].[NormalizedName]
FROM [AspNetRoles] AS [a]
WHERE [a].[NormalizedName] = @__normalizedRoleName_0
2023-02-27 23:53:36.935 -05:00 [INF] Executing ObjectResult, writing value of type 'allopromo.Core.Domain.ApplicationUser'.
2023-02-27 23:53:37.000 -05:00 [INF] Executed action allopromo.Controllers.UserController.CreateUser (allopromo.Api) in 132335.6688ms
2023-02-27 23:53:37.001 -05:00 [INF] Executed endpoint 'allopromo.Controllers.UserController.CreateUser (allopromo.Api)'
2023-02-27 23:53:37.006 -05:00 [INF] Request finished in 132416.1369ms 200 application/json; charset=utf-8
2023-02-27 23:53:37.629 -05:00 [ERR] Failed executing DbCommand (96,442ms) [Parameters=[@__p_0='?' (Size = 450), @__p_1='?' (Size = 450)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [a].[UserId], [a].[RoleId], [a].[Discriminator], [a].[RoleId1], [a].[UserId1]
FROM [AspNetUserRoles] AS [a]
WHERE ([a].[UserId] = @__p_0) AND ([a].[RoleId] = @__p_1)
2023-02-27 23:53:38.619 -05:00 [ERR] An exception occurred while iterating over the results of a query for context type 'allopromo.Infrastructure.Data.AppDbContext'.
Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid column name 'Discriminator'.
Invalid column name 'RoleId1'.
Invalid column name 'UserId1'.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__169_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:01f77780-50bb-42e6-aa75-6c4991a0d985
Error Number:207,State:1,Class:16
Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid column name 'Discriminator'.
Invalid column name 'RoleId1'.
Invalid column name 'UserId1'.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__169_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:01f77780-50bb-42e6-aa75-6c4991a0d985
Error Number:207,State:1,Class:16
